src/components/Avatar/connector.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux'
+import { showPopoverHandler } from '../../actions'
+
+export function mapDispatchToProps (dispatch) {
+  return {
+    onMouseOver: showPopoverHandler(dispatch)
```

Comment:
the naming feels a bit off here -- calling it `onMouseOver` suggests to me that it will always be the mouseover behavior, but it's controlled by the `showPopover` bool. maybe just keep calling it `showPopoverHandler` through the component?